server/.eslintrc.js

```diff
@@ -3,37 +3,7 @@ module.exports = {
     "ecmaVersion": 6,
     "sourceType": "script"
   },
-  "extends": "airbnb-base",
   "plugins": [
-    "dependencies",
-  ],
-  "rules": {
```

Comment:
Whoops... Server side eslint rules have disappeared completely. Was that intentional?

Reply:
Yes, it was intentional. I put them back as I am happy with the relaxed rules on the client.